### PR TITLE
Make F5 restart the Chrome app

### PR DESCRIPTION
**Pull request by alancutter**
Fixes https://github.com/shans/chromez/issues/23.

### CODE REVIEW

File: app/main.js [hunk 1]
```diff
@@ -1 +1,8 @@
 chrome.power.requestKeepAwake("display");
+
+window.addEventListener('keydown', event => {
```
Comment (shans) [inline app/main.js:3]: Technically if you want to replicate normal reload functionality this should be keyup

Comment (alancutter) [inline app/main.js:3]: Just tested locally, it seems to be on key down.

File: app/main.js [hunk 2]
```diff
@@ -1 +1,8 @@
...
+  // F5
+  if (event.keyCode == 116) {
+    webview.reload();
```
Comment (dstoc) [inline app/main.js]: Could we make this restart the whole app instead (to pick up new versions of the app)?

I think it's `chrome.runtime.reload();`

Comment (alancutter) [inline app/main.js]: Done.

**Pull request merged**